source/docs/articles/sites/domains/switching-dns.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Switching DNS From One Pantheon Site to Another
+description: Learn how to change DNS details between Pantheon sites.
+categories:
+  - sites
+tags:
+  - domains
+---
+There are some scenarios (e.g. a redesigned site) where you may want to switch a domain's DNS between Pantheon sites. While each environment can configure its own custom domain, the same domain cannot be added to more than one environment. However, it's possible to point a domain to another site with minimal impact to visitors.
+
+## Before You Begin
+If you are using the HTTPS protocol with your own certificate, [enable HTTPS](/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication/) on the new site before proceeding.
+
+Also, check `settings.php` or `wp-config.php` and comment out any hardcoded redirect logic for Pantheon's Platform domains (e.g.  `live-mysite.pantheon.io`). Once DNS has been configured, you can update the redirects to use the default Platform domain (e.g. `live-mysite.pantheonsite.io`) if needed.
+
+## Determine the New DNS Records
+Since the same domain cannot be added to more than one environment, you'll need to use a temporary domain name to determine the DNS records. If the launched site has `example.com` configured, temporarily add something like `example-new.com` to the new site's Live environment and click **Show recommended DNS records** for both the bare domain and wwww.
```

Comment:
What's the advantage of determining the new DNS records using a temporary domain like example-new.com ? The instructions below do not utilize the values determined here, feels like we're missing a tip maybe?